templates/admin/subscribers.html

```diff
@@ -45,22 +45,22 @@ <h3 class="channels-title" title="Select a version to only show features for tha
 
 <!-- <div id="channels" class="layout horizontal center">
   <h3>Filter by milestone:</h3>
-  <div class="chrome_version layout horizontal center chrome_version--stable" title="Current Stable">
+  <div class="chrome_version layout horizontal center chrome_version- -stable" title="Current Stable">
```

Comment:
This whole block of code from line 46 to line 68 has been commented out for a while.  I think you could just delete the whole block.

Reply:
done